source/System.h

```diff
@@ -197,6 +208,8 @@ class System {
 	std::vector<Asteroid> asteroids;
 	const Sprite *haze = nullptr;
 	std::vector<RandomEvent<Fleet>> fleets;
+	std::vector<Counter> fleetCounters;
```

Comment:
Any reason this can't be an `std::map<int, int>`?

Reply:
Ah, no

Reply:
I will do this tomorrow

Reply:
No wait I am doing it now

Reply:
done

Reply:
I just like structs